NetSDK/Domain/KeyImpression.cs

```diff
@@ -11,6 +7,8 @@ public class KeyImpression
         public string keyName { get; set; }
         public string treatment { get; set; }
         public long time { get; set; }
+        public long? changeNumber { get; set; }
```

Comment:
To allow nulls, for example when there is no split and we return "control" and "rules not found" as a label